### ISSUE TITLE
EnumVariantNames does not respect the "serialize" property

### ISSUE THREAD

**Issue by apyrgio** [Peternator7/strum#93]
If we define an enum with a variant that has a special way of being serialized, e.g.:

```rust
#[derive(EnumVariantNames)]
enum Fruit {
    Apple,
    #[strum(serialize = "tasty_orange")]
    Orange
}
```

it seems that it is not serialized properly. In this example, we get the following:

```rust
assert_eq!(Fruit::VARIANTS, ["Apple", "Orange"]);
```

Is this the intended behavior, or have I used it incorrectly?

**Comment by johanneskoester** [comment]
Same issue here. It seems to work with serialize_all, but not with individual overrides.

**Comment by Peternator7** [comment]
Should be fixed in `0.19.1`
